### PR TITLE
Update pt_BR.json

### PR DESCRIPTION
**Pull request by fbrisola**
Correct typos and names to common brazilian portuguese.

### CODE REVIEW

File: locales/pt_BR/pt_BR.json [hunk 1]
```diff
@@ -654,7 +654,7 @@
     "United States Minor Outlying Islands": "Ilhas Menores Distantes dos Estados Unidos",
     "United States Outlying Islands": "Ilhas Distantes dos Estados Unidos",
     "Update": "Atualizar",
-    "Update & Continue Editing": "Atualizar E Continuar A Edição",
+    "Update & Continue Editing": "Atualizar e Continuar \ Edição",
```
Comment (andrey-helldar) [inline locales/pt_BR/pt_BR.json:657]: Are you sure you need to escape the space character?

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: locales/pt_BR/pt_BR.json (hunk 1 above)
Comment (fbrisola) [reply locales/pt_BR/pt_BR.json:657]: Sorry for the fat finger. The correct is "Update & Continue Editing": "Atualizar e Continuar a Edição"
I will fix it now.

**Review by andrey-helldar — APPROVED** [review]

**Comment by andrey-helldar** [comment]
Obrigado

**Comment by fbrisola** [comment]
Glad to help!

**Pull request merged**